util.c

```diff
@@ -63,6 +63,16 @@ static log_function_type *current_log_function = log_file;
 static FILE *current_log_file = NULL;
 int log_time_asc = 1;
 
+#ifdef USE_LOG_PROCESS_ROLE
+static const char *global_process_role = NULL;
```

Comment:
Please don't introduce more static global variables.

Reply:
Done in commit 5a4b474a